Adadie/Fresh
# #21-dashboard page

#### What does this PR do?
This PR will update the dashboard page by adding sections, buttons, changing style and adding links to buttons that direct to the page.
#### Description of Task to be completed?
The dashboard should contain the sections where the user can:
- View a list of their articles
- Create a new article
- Update an existing article
- Have a delete button for an existing article 
- Update their profile information
- Check Reminder settings.
- View Number of entries since registration.
#### How should this be manually tested? 
First clone this repository to your local PC [Click here to get the link]( https://github.com/Junior-Adadie/Fresh.git)
Open index.html file with your browser.
Make sure your browser is up to date.
Check if the profile icon button leads to the login page
Click submit to enter the post-login page/dashboard
Check if the tasks mentioned above work properly
#### What are the relevant Trello stories?
[Trello Card](https://trello.com/c/ot2afUHf/21-dashboard)
#### Screenshots (if appropriate)
![Capture2](https://user-images.githubusercontent.com/68109866/88717165-ca3b4380-d128-11ea-9b69-820bb355689c.PNG)
![Capture3](https://user-images.githubusercontent.com/68109866/88717174-cc050700-d128-11ea-9e01-9277b7424683.PNG)
![Capture4](https://user-images.githubusercontent.com/68109866/88717179-cd363400-d128-11ea-9152-543195ddccdc.PNG)
